tools/amend-event-data.js

```diff
@@ -22,6 +22,7 @@ const trees = {
   idb: ["IDBDatabase", "IDBTransaction", "IDBRequest"],
   bt: ['Bluetooth', 'BluetoothDevice', 'BluetoothRemoteGATTService',
        'BluetoothRemoteGATTCharacteristic', 'BluetoothRemoteGATTDescriptor'],
+  serial: ['Serial', 'SerialPort'],
```

Comment:
This makes me realize that the list needs to be maintained in two places (here and in tests), which isn't ideal.

Also, I'm not crazy about using a RegExp for element interfaces, since that means relying on a naming convention. Typically, I don't think `CSSPseudoElement` should be viewed as being part of a tree. Or should it? At least, it does not inherit from `Element` or `Node`.

Since the script already reuses the data in the `idlparsed` extracts, we should be able to follow the inheritance chain instead as done in the tests.

I'll update the script to remove the duplication and rather follow the inheritance chain.